### ISSUE TITLE
Screen reader users can access the report pane before a call has ended

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/call.zetk.in#264]
## Steps to reproduce

1. Go to the call page in a window of 600px width or greater.
2. Start VoiceOver.
3. Use `^⌥→` to move focus beyond the end of the input pane.

## Expected outcome

No more panes are possible to access using a mouse and keyboard so none should be available in VoiceOver either. 

## Actual outcome

It's possible to interact with the report pane in VoiceOver during the call.

![macOS VoiceOver screen recording where we escape into a hidden part of the UI](https://user-images.githubusercontent.com/566159/192093330-383d6131-fede-4814-b79b-cd01ade66507.gif)

As the report pane only makes sense to access after a call has concluded, being able to access it with assistive technology during an ongoing call falls under [WCAG 2.4.3 Focus Order](https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html) because the focus order doesn't preserve meaning. [WCAG 1.3.1 Info and Relationships](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html) may also be applicable as the hidden state of the report pane's presentation isn't available programmatically to assistive technology.

## Recommendation

It seems like this UI is rendered before it's needed to facilitate the nice slide transition when it appears. One option could be to live without the animation. Another could be to try setting the pane to [`inert`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert) until it appears.

This issue is effectively a duplicate of https://github.com/zetkin/call.zetk.in/issues/104, as this is the same bug reported there except with a slightly wider scope. The fix for that original issue was to disable `tab` keypresses entirely, and fixing https://github.com/zetkin/call.zetk.in/issues/263 will walk back that original fix. Addressing this issue in a way that covers screen reader usage as well as plain old tab navigation will unblock implementing the recommendations for https://github.com/zetkin/call.zetk.in/issues/263.
